src/test/java/com/hubspot/jinjava/lib/exptest/isDivisibleByExpTestTest.java

```diff
@@ -0,0 +1,78 @@
+package com.hubspot.jinjava.lib.exptest;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertEquals;
+
+import com.hubspot.jinjava.Jinjava;
+import java.util.HashMap;
+import org.junit.Before;
+import org.junit.Test;
+
+public class isDivisibleByExpTestTest {
+  private static final String DIVISIBLE_BY_TEMPLATE = "{{ %s is divisibleby %s }}";
+
+  private Jinjava jinjava;
+
+  @Before
+  public void setup() {
+    jinjava = new Jinjava();
+  }
+
+  @Test
```

Comment:
Might be good to add at least one positive test case as well to confirm we didn't break anything.

Reply:
I'll break out the one positive test on line 59 (currently embedded in a negative test) to be its own test